README.md

```diff
@@ -2,53 +2,25 @@
 
 # Hello Microverse
 
-> This project of hello microverse project about github flow.
-
-![screenshot](./app_screenshot.png)
+> This is a basic project called Hello Microverse about basic github flow.
 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+- HTML, CSS
+- Git , Github, git Actions
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
 
 
 
 ## Authors
 
-👤 **Author1**
+👤 **Houssam HICHAMI**
 
 - GitHub: [@githubhandle](https://github.com/redwing555)
 
 
```

Comment:
- [ ] **OPTIONAL:** When I look through your comments, I see that they are not written in a professional way, in the next projects try to begin all your comments with capital letters, also instead of writing `
adding node_modules file to .gitignore`, use present simple tense `
Add node_modules file to .gitignore`(for all that like comments).